### PR TITLE
feat: tags build stage

### PR DESCRIPTION
**Pull request by gelleson**
Added field tags for forwarding at the build stageю

### CODE REVIEW

**Comment by gelleson** [comment]
it's the same pull request but more clean.

File: cmd/helpers.go [hunk 1]
```diff
@@ -143,6 +143,7 @@ func BuildDocker(binaryName string, buildMode string, projectOptions *ProjectOpt
 		"-v", fmt.Sprintf("%s:/build", filepath.Join(fs.Cwd(), "build")),
 		"-v", fmt.Sprintf("%s:/source", fs.Cwd()),
 		"-e", fmt.Sprintf("LOCAL_USER_ID=%v", userid),
+		"-e", fmt.Sprintf("FLAG_TAGS=\"%s\"", projectOptions.Tags),
```
Comment (leaanthony) [inline cmd/helpers.go:146]: I don't think the escaped quotes are needed here from what I can see https://github.com/karalabe/xgo/blob/c5ccff8648a77686d274c1984e1cd6319736f5cb/xgo.go#L297

**Review by leaanthony — CHANGES_REQUESTED** [review]

File: cmd/helpers.go (hunk 1 above)
Comment (gelleson) [reply cmd/helpers.go]: I have fixed it :)

**Comment by leaanthony** [comment]
Thanks!

**Pull request merged**